content/en/docs/ops/best-practices/security/index.md

```diff
@@ -42,6 +42,80 @@ For example, in the [authorization for HTTP traffic task](/docs/tasks/security/a
 the authorization policy named `allow-nothing` makes sure all traffic is denied by default.
 From there, other authorization policies allow traffic based on specific conditions.
 
+### Understand path normalization in authorization policy
+
+The enforcement point of authorization policy is the Envoy proxy which is different from the actual resource access point that is
+usually in the backend application. A policy mismatch happens when the Envoy proxy and the backend application interprets the request
+differently.
+
+The mismatch leads to either unexpected rejection or policy bypass. The latter is usually a security incident that needs to be
+fixed immediately, and it's also why we need path normalization in the authorization policy.
+
+For example, you have an authorization policy to reject requests with path `/data/secret`. A request with path `/data//secret` will
+not be rejected because it does not match with the path defined in the authorization policy due to the extra forward slash `/` in the path.
```

Comment:
```suggestion
not be rejected because it does not match the path defined in the authorization policy due to the extra forward slash `/` in the path.
```